libs/utils/fs_test.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestExists(t *testing.T) {
+	// Create a temporary file for testing
+	tmpFile, err := os.CreateTemp("", "test")
```

Comment:
i think i'd prefer if we use t.TempDir() here please https://pkg.go.dev/testing#T.TempDir